hw/bsp/nrf52dk/pkg.yml

```diff
@@ -39,8 +39,8 @@ pkg.cflags:
     - '-DPDM_ENABLED=0'
     - '-DPERIPHERAL_RESOURCE_SHARING_ENABLED=1'
     - '-DPWM0_ENABLED=1'
-    - '-DPWM1_ENABLED=0'
-    - '-DPWM2_ENABLED=0'
+    - '-DPWM1_ENABLED=1'
+    - '-DPWM2_ENABLED=1'
```

Comment:
Partially changed as requested. Only changed PWM1_ENABLED and PWM2_ENABLED values to 0. Waiting for further instructions/requests on this.